.travis.yml

```diff
@@ -20,6 +20,7 @@ install:
   - sudo apt-get install -y gdb
 
 script:
+  - export AWS_KVS_LOG_LEVEL=2
```

Comment:
Do we need this?

Reply:
This is the debug level. We can get some info from the runs.